packages/providers/audit-logs-local/src/index.ts

```diff
@@ -1,43 +1,53 @@
-'use strict';
-
-const auditLogContentType = require('./content-types/audit-log');
-
-const provider = {
-  async register({ strapi }) {
+import type { Strapi } from '@strapi/strapi';
+import { schema as auditLogContentType } from './content-types/audit-log';
+
+interface Event {
+  action: string;
+  date: Date;
+  userId: string | number;
+  payload: Record<string, unknown>;
+}
```

Comment:
```suggestion
interface Event<T = unknown> {
  action: string;
  date: Date;
  userId: string | number;
  payload: Record<string, T>;
}
```

wdyt?

Reply:
I would probably just pass the payload type in that case. but I don't want to do this in v4 for now. just moving to ts a simple as possible to avoid conflicts when we really do clearn typescript typings